js/state.js

```diff
@@ -0,0 +1,25 @@
+const form = document.querySelector('.ad-form');
+const header = document.querySelector('.ad-form-header');
+const fieldsetS = document.querySelectorAll('.ad-form__element');
+
+const filters = document.querySelector('.map__filters');
+const features = document.querySelector('#housing-features');
+const type = document.querySelector('#housing-type');
+const price = document.querySelector('#housing-price');
+const rooms = document.querySelector('#housing-rooms');
+const guests = document.querySelector('#housing-guests');
+
+const disabledForm = ()=>{
+  form.classList.add('ad-form--disabled');
+  filters.classList.add('ad-form--disabled');
+  header.setAttribute('disabled', 'disabled');
+  features.setAttribute('disabled', 'disabled');
+  type.setAttribute('disabled', 'disabled');
+  price.setAttribute('disabled', 'disabled');
+  rooms.setAttribute('disabled', 'disabled');
+  guests.setAttribute('disabled', 'disabled');
+  fieldsetS.forEach((elem)=>{
+    elem.setAttribute('disabled', 'disabled');
+  });
+};
+export {disabledForm};
```

Comment:
```suggestion
const adForm = document.querySelector('.ad-form');
const mapFilter = document.querySelector('.map__filters');
const adFormFieldsets = adForm.querySelectorAll('fieldset');
const mapFilterFieldsets = mapFilter.querySelectorAll('fieldset');
const mapFilterSelects = mapFilter.querySelectorAll('select');

const toggleFormDisabled = (isDisabled) => {
  adForm.classList.toggle('ad-form--disabled', isDisabled);
  mapFilter.classList.toggle('map__filters--disabled', isDisabled);
  adFormFieldsets.forEach((item) => {
    item.disabled = isDisabled;
  });
  mapFilterSelects.forEach((item) => {
    item.disabled = isDisabled;
  });
  mapFilterFieldsets.forEach((item) => {
    item.disabled = isDisabled;
  });
};

export {toggleFormDisabled};
```
в целом валидно, но на каждый элемент писать, это долго. Попробуй так оптимизировать